pkg/connector/ble/iface/adapter.go

```diff
@@ -0,0 +1,14 @@
+package iface
```

Comment:
To me it seems more natural to put the shared code in `pkg/connector/ble` rather than a separate `iface` directory. Any objections to moving it back?

Reply:
yes I had it like that first. @Lenart12 suggested using go-ble as a default, but the problem was cyclic import.
eg. go-ble wants to use `ScanResult` from `pkg/connector/ble` and `pkg/connector/ble` wanted to set `go-ble` as a default impl.

I am using this as a library and I didn't notice any breaking change. I was able to compile without a change.

I can move it back, but then library user will have to pick the impl